sources/packages/backend/libs/test-utils/src/factories/disbursement-receipt.ts

```diff
@@ -53,3 +60,87 @@ export function createFakeDisbursementReceipt(relations: {
   receipt.createdAt = now;
   return receipt;
 }
+
+/**
+ * Create federal(FE) and provincial(BC) receipts records based in the disbursement and its awards values.
+ * Each disbursement will generate two records on sims.disbursement_receipts, one for federal and another
+ * for provincial awards.
+ * @param disbursementSchedule disbursement with the awards to have the receipts created.
+ * @returns returns the federal and provincial receipt created and their awards receipts.
+ */
+export async function saveFakeDisbursementReceiptsFromDisbursementSchedule(
+  db: E2EDataSources,
```

Comment:
Minor. This param is missing in the comment.

Reply:
Added following other E2E tests.